perf_dashboard/benchmarks/views.py

```diff
@@ -868,21 +928,29 @@ def get_latency_vs_conn_y_series(df, telemetry_mode, quantiles):
     y_series_data = []
     for thread in [2, 4, 8, 16, 32, 64]:
         data = df.query('ActualQPS == 1000 and NumThreads == @thread and Labels.str.endswith(@telemetry_mode)')
-        if not data[quantiles].head().empty:
-            y_series_data.append(data[quantiles].head(1).values[0] / 1000)
-        else:
+        try:
+            data[quantiles].head().empty
+        except KeyError as e:
+            print(e)
             y_series_data.append('null')
+        else:
+            if not data[quantiles].head().empty:
+                y_series_data.append(data[quantiles].head(1).values[0] / 1000)
     return y_series_data
 
 
 def get_latency_vs_qps_y_series(df, telemetry_mode, quantiles):
     y_series_data = []
     for qps in [10, 100, 500, 1000, 2000, 3000]:
         data = df.query('ActualQPS == @qps and NumThreads == 16 and Labels.str.endswith(@telemetry_mode)')
-        if not data[quantiles].head().empty:
-            y_series_data.append(data[quantiles].head(1).values[0] / 1000)
-        else:
+        try:
+            data[quantiles].head().empty
+        except KeyError as e:
```

Comment:
probably better to just use get instead of doing try catch to avoid keyerror

Reply:
yea, can do that, but doesn't save any # of lines. Will update later